spark/common/src/main/java/org/apache/sedona/core/formatMapper/netcdfParser/SerNetCDFUtils.java

```diff
@@ -31,6 +31,8 @@
  */
 package org.apache.sedona.core.formatMapper.netcdfParser;
 
+import org.geotools.coverage.grid.GridCoverage2D;
```

Comment:
Why add these imports since they are not used anyway?

Reply:
Removed this, this was part of my original work push to netcdf branch